### PR TITLE
[SERVER] - adding server building files

### PR DESCRIPTION
**Pull request by ghost**
**Objective**
Files needed to build the smartpeak-server image.

### CODE REVIEW

**Comment by dmccloskey** [comment]
Hi Ahmed.  Do you know what is failing on the CI?

File: dockerfiles/Dockerfile.smartpeak-server [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# INSTITUTIONS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+# OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+# OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+# ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# --------------------------------------------------------------------------
+# $Maintainer: Ahmed Khalil $
```
Comment (dmccloskey) [inline dockerfiles/Dockerfile.smartpeak-server]: Please be sure to add me as a maintainer

**Review by dmccloskey — CHANGES_REQUESTED** [review]
Small detail and then good to go.

**Comment by ghost** [comment]
> Hi Ahmed. Do you know what is failing on the CI?

Hi Douglas, the latest OpenMS/Qt5::Sql wasn't applied.

**Review by bertrandboudaud — APPROVED** [review]
Nice :)

**Pull request merged**